src/include/cluster/dialogs.rb

```diff
@@ -1008,22 +1015,33 @@ def ValidateService
     end
 
     def UpdateServiceStatus
-      ret = 0
-      ret = Service.Status("pacemaker")
-      if ret == 0
+      ret_pacemaker = 0
+      ret_qdevice = 0
+      ret_pacemaker = Service.Status("pacemaker")
+      if Cluster.corosync_qdevice
+        ret_qdevice = Service.Status("corosync-qdevice")
+      end
+      if ret_pacemaker == 0 && ret_qdevice == 0
```

Comment:
I am thinking the situation:
`corosync-qdevice` service is running, user configure not using `corosync-qdevice` in YaST. Is that necessary to enable the `stop_now` and allow user to stop `corosync-qdevice` in this case?
But sure, this will cause more complex logic...

Reply:
hmmm, there would be a complex situation need to think about. Eg, pacemaker+corosync is up and configure the qdevice with Yast, at this moment, "service status" would be not running, actually, pacemaker+corosync is running.
These kinds situation happened also in booting section.
If so, start/stop(enable/disable) these three services separately seems exact.

Reply:
A easier way is show both enable/disable, start/stop of pacemaker, corosync-qdevice. Though, it needs to show 4 boxes in the page.